.circleci/config.yml

```diff
@@ -11,3 +11,93 @@ workflows:
           context: Danger
       - kitchen/lint:
           name: lint
+      - kitchen/dokken-single:
+          name: server-debian-8
+          suite: server
+          platform: debian-8
+          requires: [  danger, lint ]
+      - kitchen/dokken-single:
+          name: server-debian-9
+          suite: server
+          platform: debian-9
+          requires: [  danger, lint ]
+      - kitchen/dokken-single:
+          name: server-centos-7
+          suite: server
+          platform: centos-7
+          requires: [  danger, lint ]
+      - kitchen/dokken-single:
+          name: server-fedora-latest
```

Comment:
No fedora please ☺

Reply:
You might be better off using the dokken multi.if that doesn't fit your needs we need to change that

Reply:
Are we removing fedora from dokken-multi? sorry haven't been keeping up to date

Reply:
The newst one does. @tas50 did a survey of Chef users and most didn't user Fedora, and it was  the distro that gave us the most greif. 

That and Fedora bugs aren't fixed until they're in RHEL quite often :(